specification/cognitiveservices/data-plane/Language/stable/2022-05-01/conversationalanalysisauthoring.json

```diff
@@ -0,0 +1,3060 @@
+        },
+        "entities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationalAnalysisAuthoringConversationExportedEntity"
+          }
+        },
+        "utterances": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationalAnalysisAuthoringConversationExportedUtterance"
+          }
+        }
+      },
+      "x-ms-discriminator-value": "conversation",
+      "x-ms-client-name": "ConversationExportedProjectAssets"
+    },
+    "ConversationalAnalysisAuthoringConversationExportedUtterance": {
+      "required": [
+        "text",
+        "intent"
+      ],
+      "type": "object",
+      "properties": {
+        "entities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationalAnalysisAuthoringExportedUtteranceEntityLabel"
+          }
+        },
+        "text": {
+          "type": "string"
+        },
+        "language": {
+          "type": "string"
+        },
+        "intent": {
+          "type": "string"
+        },
+        "dataset": {
+          "description": "The dataset for this utterance. Allowed values are 'Train' and 'Test'.",
+          "type": "string"
+        }
+      },
+      "x-ms-client-name": "ConversationExportedUtterance"
+    },
+    "ConversationalAnalysisAuthoringCreateDeploymentOptions": {
+      "required": [
+        "trainedModelLabel"
+      ],
+      "type": "object",
+      "properties": {
+        "trainedModelLabel": {
+          "description": "Represents the trained model label.",
+          "type": "string"
+        }
+      },
+      "x-ms-client-name": "CreateDeploymentOptions"
+    },
+    "ConversationalAnalysisAuthoringCreateProjectOptions": {
+      "required": [
+        "projectKind",
+        "projectName",
+        "language"
+      ],
+      "type": "object",
+      "properties": {
+        "projectKind": {
+          "description": "Represents the project kind.",
+          "$ref": "#/definitions/ConversationalAnalysisAuthoringProjectKind"
+        },
+        "settings": {
+          "description": "Gets or sets the project settings.",
+          "$ref": "#/definitions/ConversationalAnalysisAuthoringProjectSettings"
+        },
+        "projectName": {
+          "description": "Gets or sets the new project name.",
+          "type": "string"
+        },
+        "multilingual": {
+          "description": "Gets or sets a value indicating whether gets or sets whether the project is multilingual or not.",
+          "type": "boolean"
+        },
+        "description": {
+          "description": "Gets or sets the new project description.",
+          "type": "string"
+        },
+        "language": {
+          "description": "Gets or sets the project language.",
+          "type": "string"
+        }
+      },
+      "x-ms-client-name": "CreateProjectOptions"
+    },
+    "ConversationalAnalysisAuthoringDeploymentJobState": {
```

Comment:
Are there any differences across job states and status? Couldn't they be folded into a single definition. IIRC, for 2022-05-15-preview someone moved all the job state models and related properties to common.json for unification.